docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/MatrixFactorization.cs

```diff
@@ -74,32 +92,44 @@ private static List<MatrixElement> GenerateMatrix()
             var dataMatrix = new List<MatrixElement>();
             for (uint i = 0; i < MatrixColumnCount; ++i)
                 for (uint j = 0; j < MatrixRowCount; ++j)
-                    dataMatrix.Add(new MatrixElement() { MatrixColumnIndex = i, MatrixRowIndex = j, Value = (i + j) % 5 });
+                    dataMatrix.Add(new MatrixElement() { MatrixColumnIndex = i,
+                        MatrixRowIndex = j, Value = (i + j) % 5 });
+
```

Comment:
Empty line. Is it produced by an auto-formatting tool?

Reply:
Actually, Zeeshan asked us to do that, so that's what we've been doing as a norm.